.github/workflows/CI.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
```

Comment:
Are we not test x86 ?  (Might be worth to add at least for one of the platforms, usually windows x86 is the most finicky).

Reply:
Although neither MAT.jl tests on x86 anymore.

Reply:
I can copy in the matrix from HDF5 to #280 since I have to rebase on top of this now.